test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -68,3 +70,350 @@ def forward(self, x):
                 self.checkGraphModuleNodes(m,
                                            expected_node_occurrence=node_occurrence,
                                            expected_node_list=node_list)
+
+    @skipIfNoX86
+    def test_conv2d_unary_with_quantizer_api(self):
```

Comment:
what does conv2d_unary mean? could you add a comment here?

Reply:
Thanks for the comment. Docstring added here and also for the other 5 UTs.